spot_rl_experiments/spot_rl/utils/robot_subscriber.py

```diff
@@ -36,6 +36,7 @@ class SpotRobotSubscriberMixin:
 
     def __init__(self, spot=None, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        rospy.loginfo(f"name - {self.node_name}")
```

Comment:
what does this class do? Can we get a class docstring?

Reply:
Actually this was a debug print. I will remove this.

Reply:
removed